### PR TITLE
Bug/hidden columns padding

### PR DESCRIPTION
**Pull request by napafundi**

### CODE REVIEW

File: app/styles/gavant-ember-table.scss [hunk 1]
```diff
@@ -13,13 +13,14 @@
...
-        }
-    }
+    //pre-ETWA: Provides space for the pan-button
+    // &.has-hidden-columns {
+    //     .table > thead > tr {
+    //         > th:last-child {
+    //             padding-right: 18px;
+    //         }
+    //     }
+    // }
```
Comment (billdami) [inline app/styles/gavant-ember-table.scss:23]: do we have a post-ETWA solution to replace this, for giving right padding to the "last visible column" for the cases where the column header text would be long enough to be covered by the pan button?

Comment (napafundi) [reply app/styles/gavant-ember-table.scss:23]: that's what would be achieved by this commented out part right? "pre-ETWA" means the way you initially created it before I had to do the visibility funkiness. Basically saying, if/when that issue gets fixed, remove any ETWA code and uncomment any pre-ETWA code.

Comment (napafundi) [reply app/styles/gavant-ember-table.scss:23]: oh oh oh oh, I see what you mean. Post-ETWA as in "right now". I don't think so, but I can add that.

Comment (billdami) [reply app/styles/gavant-ember-table.scss:23]: yep exactly lol. well i guess in the apps where this wasn't working at all (CC, PEP), did we do something in those apps (e.g. a CSS tweak or something) to resolve this, and if so should we just incorporate that fix/workaround into the addon, or was it just never encountered in those instances.

Comment (napafundi) [reply app/styles/gavant-ember-table.scss]: I haven't encountered it in CC or JR but the responsiveness of those sites is... nothing to write home about lol. But I added a fix.

**Review by billdami — APPROVED** [review]

**Pull request merged**